versioned_docs/version-1.0/language/functions.mdx

```diff
@@ -581,6 +485,103 @@ Both preconditions and postconditions are considered [`view` contexts](#view-fun
 any operations that are not legal in functions with `view` annotations are also not allowed in conditions. 
 In particular, this means that if you wish to call a function in a condition, that function must be `view`. 
 
+## View Functions
+
+Functions can be annotated as `view` to indicate that they do not modify any external state or any account state. 
+A `view` annotation can be added to the beginning of a function declaration or expression like so: 
+
+```cadence
+view access(all) fun foo(): Void {}
+let x = view fun(): Void {}
+access(all) struct S {
+    view access(all) fun foo(): Void {}
+    view init()
+}
```

Comment:
Do we support any order for `view` and access modifiers?

Anyways, having the access modifier followed by the `view` modifier looks more natural, and aligns with the practice of `access(all)` being on a separate line. 

```suggestion
access(all) 
view fun foo(): Void {}

let x = view fun(): Void {}

access(all) 
struct S {

    access(all)
    view fun foo(): Void {}
    
    view init()
}
```